test/Test/Oscoin/Data/OscoinTx.hs

```diff
@@ -113,30 +114,36 @@ propMultiTransfer Dict = property $ do
 
     -- A single invalid message (`t0`) fails the transaction
     invalidTx <- pure (mkTxPayload [t1, t0, t2] alice)
-    applyTxPayload (invalidTx { txFee = minimumTxFee invalidTx }) ws === Left (ErrInvalidTransfer 0)
+    applyTxPayload (invalidTx { txFee = minimumTxFee invalidTx }) eve ws === Left (ErrInvalidTransfer 0)
 
     -- Multiple messages combine
     tx <- pure (mkTxPayload [t1, t1, t2] alice)
-    (ws', _) <- evalEither $ applyTxPayload (tx { txFee = minimumTxFee tx }) ws
+    (ws', _) <- evalEither $ applyTxPayload (tx { txFee = minimumTxFee tx }) eve ws
 
     lookupBalance bob ws' === Right 4
 
 propTxFeeBurn :: forall c. Dict (IsCrypto c) -> Property
 propTxFeeBurn Dict = property $ do
-    alice <- publicKey @c
-    tx    <- pure (mkTxPayload [] alice)
-    bal   <- forAll genBalancePositive
-    ws    <- evalEither $ creditAccount alice bal WorldState.empty
+    (alice, eve)
```

Comment:
Having Eve, the evil eavesdropper, roaming our tests makes me slightly uncomfortable ;)

Reply:
Hahaha yeah I was thinking "what comes after bob?"

Reply:
Maybe Charlie? Or Ben the beneficiary?

Reply:
Yeah, maybe I'll go with Charlie -- using `eve` it might even be misleading, like suggesting we are testing against adversarial behaviour 😮

Reply:
Haha fair enough.

Reply:
b5384c2